homeassistant/components/here_weather/const.py

```diff
@@ -0,0 +1,393 @@
+    MODE_ASTRONOMY,
+    MODE_HOURLY,
+    MODE_DAILY,
+    MODE_DAILY_SIMPLE,
+    MODE_OBSERVATION,
+]
+DEFAULT_MODE = MODE_DAILY_SIMPLE
+
+ASTRONOMY_ATTRIBUTES: dict[str, dict[str, str | None]] = {
+    "sunrise": {"name": "Sunrise", "unit_of_measurement": None},
+    "sunset": {"name": "Sunset", "unit_of_measurement": None},
+    "moonrise": {"name": "Moonrise", "unit_of_measurement": None},
+    "moonset": {"name": "Moonset", "unit_of_measurement": None},
+    "moonPhase": {"name": "Moon Phase", "unit_of_measurement": "%"},
+    "moonPhaseDesc": {"name": "Moon Phase Description", "unit_of_measurement": None},
+    "city": {"name": "City", "unit_of_measurement": None},
+    "latitude": {"name": "Latitude", "unit_of_measurement": None},
+    "longitude": {"name": "Longitude", "unit_of_measurement": None},
+    "utcTime": {"name": "UTC Time", "unit_of_measurement": None},
+}
+
+HOURLY_ATTRIBUTES: dict[str, dict[str, str | None]] = {
+    "daylight": {"name": "Daylight", "unit_of_measurement": None},
+    "description": {"name": "Description", "unit_of_measurement": None},
+    "skyInfo": {"name": "Sky Info", "unit_of_measurement": None},
+    "skyDescription": {"name": "Sky Description", "unit_of_measurement": None},
+    "temperature": {"name": "Temperature", "unit_of_measurement": "°C"},
+    "temperatureDesc": {"name": "Temperature Description", "unit_of_measurement": None},
+    "comfort": {"name": "Comfort", "unit_of_measurement": "°C"},
+    "humidity": {"name": "Humidity", "unit_of_measurement": "%"},
+    "dewPoint": {"name": "Dew Point", "unit_of_measurement": "°C"},
+    "precipitationProbability": {
+        "name": "Precipitation Probability",
+        "unit_of_measurement": "%",
+    },
+    "precipitationDesc": {
+        "name": "Precipitation Description",
+        "unit_of_measurement": None,
+    },
+    "rainFall": {"name": "Rain Fall", "unit_of_measurement": "cm"},
+    "snowFall": {"name": "Snow Fall", "unit_of_measurement": "cm"},
+    "airInfo": {"name": "Air Info", "unit_of_measurement": None},
+    "airDescription": {"name": "Air Description", "unit_of_measurement": None},
+    "windSpeed": {"name": "Wind Speed", "unit_of_measurement": "km/h"},
+    "windDirection": {"name": "Wind Direction", "unit_of_measurement": "°"},
+    "windDesc": {"name": "Wind Description", "unit_of_measurement": "cm"},
+    "windDescShort": {"name": "Wind Description Short", "unit_of_measurement": "cm"},
+    "visibility": {"name": "Visibility", "unit_of_measurement": "km"},
+    "icon": {"name": "Icon", "unit_of_measurement": None},
+    "iconName": {"name": "Icon Name", "unit_of_measurement": None},
+    "iconLink": {"name": "Icon Link", "unit_of_measurement": None},
+    "dayOfWeek": {"name": "Day of Week", "unit_of_measurement": None},
+    "weekday": {"name": "Week Day", "unit_of_measurement": None},
+    "utcTime": {"name": "UTC Time", "unit_of_measurement": None},
+    "localTime": {"name": "Local Time", "unit_of_measurement": None},
```

Comment:
Times in the state machine must always be UTC time. I suggest removing the local time sensors.

Reply:
Done in https://github.com/home-assistant/core/pull/28910/commits/76f1ce5ef37aa3d1703900111f297761ef188cc1